packages/smooth_app/lib/pages/crop_page.dart

```diff
@@ -176,22 +176,26 @@ class _CropPageState extends State<CropPage> {
                   mainAxisAlignment: MainAxisAlignment.spaceBetween,
                   crossAxisAlignment: CrossAxisAlignment.center,
                   children: <Widget>[
-                    Row(
-                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                      children: <Widget>[
-                        _IconButton(
-                          iconData: Icons.rotate_90_degrees_ccw_outlined,
-                          onPressed: () => setState(
-                            () => _controller.rotateLeft(),
+                    Padding(
+                      padding:
+                          const EdgeInsetsDirectional.only(top: SMALL_SPACE),
```

Comment:
Not sure it's a good idea to add a padding there, as it means less space for the image, that is already not that big.

Reply:
We talk about the Crop page here, not the viewer.
Are you sure it's that small?

Reply:
We do talk about the crop page.
The image is quite small on small devices, especially given that our crop tool does not zoom in.
You're right, there is no problem on big screens.

Reply:
Is there a threshold we already use to say small screens?